packages/strapi-plugin-users-permissions/models/User.settings.json

```diff
@@ -52,6 +52,11 @@
       "via": "users",
       "plugin": "users-permissions",
       "configurable": false
+    },
```

Comment:
Have we to keep this attribute by default?
I'm ok but I would like the @Aurelsicoko feedback.
In any case it could be updated after having merge this PR.

Reply:
I need to delete this my bad